api/src/controllers/cle/classe.js

```diff
@@ -52,17 +52,21 @@ router.post("/", passport.authenticate("referent", { session: false, failWithErr
       const defaultCleCohort = await CohortModel.findOne({ name: value.cohort }, { session });
       if (!defaultCleCohort) return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND, message: "Cohort not found." });
 
-      classe = await ClasseModel.create({
-        ...value,
-        cohort: defaultCleCohort.name,
-        uniqueKey,
-        uniqueId: value.uniqueId,
-        referentClasseIds: [referent._id],
-        etablissementId: etablissement._id,
-      }, { session });
+      classe = await ClasseModel.create(
+        {
+          ...value,
+          cohort: defaultCleCohort.name,
+          uniqueKey,
+          uniqueId: value.uniqueId,
+          uniqueIdAndKey: uniqueKey + "_" + value.uniqueId,
```

Comment:
Comme `uniqueKey` est avant `uniqueId` je mettrais plutôt ça

```suggestion
          uniqueKeyAndId: uniqueKey + "_" + value.uniqueId,
```